packages/form-core/test-suites/FormatMixin.suite.js

```diff
@@ -367,7 +367,7 @@ export function runFormatMixinSuite(customConfig) {
 
         const el = /** @type {FormatClass} */ (await fixture(html`
           <${elem} .formatter=${formatterSpy}>
-            <input slot="input" value="${generatedViewValue}">
+            <input slot="input" .value="${generatedViewValue}">
```

Comment:
I think it's fine in this particular pr, but in general we should be careful with blindly replacing props and attrs.
Especially for the input, `[value]` is only synced at the initialization of a component (future attr updates don't take effect).

There's also slight discrepancies in behavior in LitElement: https://github.com/ing-bank/lion/issues/1211

Reply:
Yeah I would argue we should use this rule to "warn", not "error", because sometimes it's actually intended and totally fine to set an attribute in your template to a JS variable. For `.value` it works because value is an actual property on HTMLInputElement, but that's a coincidence. If you did it for `.foo` you would not see attribute foo in the DOM.

Updated the PR

Reply:
Oh wait this linting rule only applies to "value" attribute, never mind, then I think the rule makes sense.